neon_homeassistant_skill/__init__.py

```diff
@@ -50,6 +50,35 @@ def initialize(self):
             self.handle_set_light_color_response,
         )
 
+        resp = self.bus.wait_for_response(Message("ovos.phal.plugin.homeassistant.check_connected"))
+        if resp and resp.data.get("connected"):
+            self.log.debug("PHAL plugin connected to HA")
+            return
+        if not resp:
+            self.log.error("Home Assistant PHAL plugin not installed or not running!")
+            self.disable_unused_intents()
+            return
+
+        self.log.info("PHAL Plugin not connected to HomeAssistant")
+        self.disable_unused_intents()
```

Comment:
This should to in a method registered to `mycroft.ready` to make sure the plugin is loaded before checking. Also, refactoring to `message.forward` there should resolve [this test failure](https://github.com/mikejgray/neon-homeassistant-skill/actions/runs/5959445047/job/16165196345?pr=27#step:6:29)